test/lib/mayaUsd/fileio/testSwitchMayaReference.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2023 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import fixturesUtils
+
+import mayaUsd
+import mayaUtils
+import ufeUtils
+import usdUtils
+import testUtils
+
+from pxr import Tf, Usd, Kind, Sdf
+
+from maya import cmds
+import maya.mel as mel
+from maya import standalone
+from maya.api import OpenMaya as om
+
+import mayaUsdAddMayaReference
+import mayaUsdMayaReferenceUtils as mayaRefUtils
+from mayaUsd.lib import cacheToUsd
+
+import ufe
+
+import os, unittest
+
+class SwitchMayaReferenceTestCase(unittest.TestCase):
+    '''Test Switching Maya Reference when switching variants.
+    '''
+    pluginsLoaded = False
+    mayaSceneStr = None
+    stage = None
+    kDefaultNamespace = 'simpleSphere'
+
+    @classmethod
+    def setUpClass(cls):
+        fixturesUtils.setUpClass(__file__)
+
+    @classmethod
+    def tearDownClass(cls):
+        standalone.uninitialize()
+
+    def getCacheFileName(self):
+        return 'testSwitchMayaRefCache.usda'
+
+    def removeCacheFile(self):
+        '''
+        Remove the cache file if it exists. Ignore error if it does not exists.
+        '''
+        try:
+            os.remove(self.getCacheFileName())
+        except:
+            pass
```

Comment:
Can you fix this so there is no Chorus warning? I think you just need to do `except Exception:` instead. I also found 2-3 other places identical to this (old code) that would be great if you could clean those up as well.